script/Ciel/Day 8/08_01_CIEL08_1B.txt

```diff
@@ -168,37 +193,40 @@ Although Arihiko, the one who said it to me, is someone who leads a life that go
 [sha:70331bd1441ed20b9befad35350e3c9800f049e5]{
 -- Page 48, Offset 18380.
 -- ―――もちろん、遠野家の長男におかしな遊びを教えていないかどうか、じっくりと詰問させていただくのを前提で」
-―――Of course, I reckon that I will be thoroughly questioning him on whether he's been putting mischievous thoughts into the eldest son of the Tohno family's head."
+―――So that I can use the opportunity to thoroughly interrogate him on whether he's been putting any mischievous ideas into the head of the eldest son of the Tohno family, of course."
 }
 [sha:104dcffdc8297eafbe238500093b809e7c625fb1]{
 -- Page 48, Offset 18381.
 -- ……今さら言うまでもないけど。怖いよね、うちの妹。
-Though it goes without saying... My little sister is scary.
+It should go without saying at this point, but...
+ My little sister sure is scary.
 }
 [sha:fd76d1adba6412e11e8a27588eec36ae7353c3a1]{
 -- Page 49, Offset 18382.
 -- 秋葉と雑談を交わしていると、琥珀さんがほうじ茶を運んできてくれた。
-As I was chatting with Akiha, Kohaku-san comes in, bringing the hojicha.
+While I was chatting with Akiha, Kohaku-san returned with the houjicha.
 }
 [sha:7a9be1ecd6a088b9b9f2f13e568526ffc3f445dc]{
 -- Page 49, Offset 18383.
 -- 渋みはあるものの清涼感のあるほうじ茶の熱さが、食後のけだるさを払拭してくれる。
-The heat of the hojicha, which is refreshing though astringent, dispels my after-meal languor.
+The heat of the houjicha, which is astringent yet refreshing, dispels my after-meal languor.
```

Comment:
"astringent, yet refreshing,"

"langour" is a stupid word. "lethargy" would be my choice